recipes/xarray_leaflet/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xarray_leaflet" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "5f68b61633392404830c6abebe5f31733a87a25bb3eb5cb10d1b1b2491dec38f" %}
+{% set build = 0 %}
```

Comment:
Please only use Jinja for name and version
```suggestion
```

Reply:
Thanks, I didn't know.

Reply:
These are just some tricks to ensure that the autotick-bot has an easier job ;)